helix-term/src/commands.rs

```diff
@@ -1715,7 +1806,9 @@ fn command_mode(cx: &mut Context) {
             }
 
             if let Some(cmd) = cmd::COMMANDS.get(parts[0]) {
-                (cmd.fun)(cx, &parts[1..], event);
+                if let Err(e) = (cmd.fun)(cx, &parts[1..], event) {
+                    cx.editor.set_error(format!("{}", e));
```

Comment:
Maybe we could just `error!` here as well so that the log file contains everything at anytime. That way you also remove the `error!` calls in `paste_clipboard_impl` and so on.
I'm not sure that's always the right level though, but `error!` sounds about right as far as I know.